README.md

```diff
@@ -19,12 +21,10 @@ within a specified VPC.
 
 ### Project origin
 
-This program was developed for the [Romana project](http://romana.io), in order
-to seamlessly deploy large [Kubernetes](https://kubernetes.io) clusters across
-multiple availability zones in an Amazon VPC. While specifically designed to
-scratch our itch for this usage scenario in the context of Romana and
-Kubernetes, the vpc-router does not depend on either project and can also be
-used stand-alone.
+This program was developed for the [Romana project](http://romana.io) to 
+overcome the limit imposed on VPC route table entries (50 by default), which limits the size of clusters.  Avoiding this limit typically required running an overlay network, which does not offer the performance and visibility of native VPC networking. vpc-router also allows Romana to apply network policy to clusters that span multiple Availability Zones (AZs) without an overlay. 
```

Comment:
I know what you're trying to say, but it sounds like you're implying that vpc-router has something to do with network policies, which it does not.